variables.tf

```diff
@@ -444,3 +444,16 @@ variable "system_schema_name" {
   type        = string
   default     = "apiary_system"
 }
+
+
+variable "init_container_image" {
+  description = "Docker image for running HMS init container."
+  type        = string
+  default     = "tbd"
+}
+
+variable "init_container_version" {
+  description = "Docker image version for running HMS init container."
+  type        = string
+  default     = "0.0.1"
+}
```

Comment:
@barnharts4 @rpoluri 
These changes should probably go first before this is merged so we can set the defaults here to something thats not TBD: https://github.com/ExpediaGroup/apiary-metastore-docker/pull/78